sqldb/sqlc/queries/amp_invoices.sql

```diff
@@ -0,0 +1,79 @@
+-- name: InsertAMPInvoicePayment :exec
+INSERT INTO amp_invoice_payments (
+    set_id, state, created_at, settled_index, invoice_id
+) VALUES (
+    $1, $2, $3, $4, $5
+); 
+
+-- name: SelectAMPInvoicePayments :many
+SELECT aip.*, ip.*
+FROM amp_invoice_payments aip LEFT JOIN invoice_payments ip ON aip.settled_index = ip.id
```

Comment:
Yes, sql generates a struct where fields in both tables get a standard name, fore example:

```
type SelectAMPInvoicePaymentsRow struct {
	SetID          []byte
	State          int16
	CreatedAt      time.Time
	SettledIndex   sql.NullInt32
	InvoiceID      int32
	ID             sql.NullInt32
	SettledAt      sql.NullTime
	AmountPaidMsat sql.NullInt64
	InvoiceID_2    sql.NullInt32
}
```


We could "avoid" the overlap and specify a `.*` for one of the tables and only the distinct columns in the other table. The problem is that next time that we add a column to the second table, we need to update this query too. Because it is not much data, I think it's better to leave it as is.

Reply:
Sound good,  now I understand thanks.